[gamemodes]/[play]/play/play_config.lua

```diff
@@ -0,0 +1,104 @@
+	{457, -686.92352294922, 966.03717041016, 11.767498970032, 89.163909912109},
+	{469, 2526.2607421875, -1677.2861328125, 19.913991928101, 91.994018554688},
+	{468, 2508.7177734375, -1680.568359375, 13.305373191833, 47.999267578125},
+	{468, 2506.615234375, -1682.3544921875, 13.305373191833, 47.999267578125},
+	{468, 2504.4423828125, -1684.208984375, 13.305373191833, 47.999267578125},
+	{557, 2512.4736328125, -1675.9833984375, 13.937532424927, 89.983520507813},
+	{557, 2512.6044921875, -1669.4228515625, 13.869720458984, 90.005493164063},
+	{402, 2473.427734375, -1692.513671875, 13.326358795166, 0},
+	{568, 2466.70703125, -1687.779296875, 13.489582061768, 272.4938659668},
+	{457, 2505.09765625, -1651.40625, 13.352488517761, 135.99975585938},
+	{475, 2459.9892578125, -1670.1943359375, 13.249677658081, 0},
+	{541, 2452.68359375, -1670.029296875, 13.199068069458, 0},
+	{429, 2447.8662109375, -1669.93359375, 13.184649467468, 359.99447631836},
+	{429, 2443.0263671875, -1669.81640625, 13.260643005371, 359.99447631836},
+	{557, -2398.5610351563, -611.85290527344, 133.04846191406, 36},
+	{557, -2393.6850585938, -608.3828125, 133.04846191406, 35.996704101563},
+	{402, -2393.4907226563, -599.73876953125, 132.58843994141, 125.5},
+	{568, -2396.79296875, -594.94836425781, 132.74308776855, 125.49682617188},
+	{457, -2407.7419433594, -577.97015380859, 132.47640991211, 126.00003051758},
+	{475, -2403.4738769531, -585.46215820313, 132.54843139648, 126.00003051758},
+	{541, -2399.7482910156, -590.07781982422, 132.34843444824, 123.99987792969},
+	{429, -2416.1259765625, -589.61535644531, 132.3984375, 213.99717712402},
+	{429, -2411.0979003906, -586.19421386719, 132.3984375, 213.99717712402},
+	{468, -2389.4606933594, -604.095703125, 132.53031921387, 123.99996948242},
+	{468, -2388.0568847656, -606.24896240234, 132.53031921387, 123.99719238281},
+	{468, -2386.5554199219, -608.55224609375, 132.53031921387, 123.99719238281},
+	{469, -2444.1838378906, -604.26513671875, 132.51147460938, 300.00003051758},
+	{468, 2030.5123291016, 1557.0339355469, 10.580312728882, 272},
+	{468, 2030.7028808594, 1559.1748046875, 10.580312728882, 271.99951171875},
+	{468, 2031.1046142578, 1561.4478759766, 10.580312728882, 271.99951171875},
+	{557, 2025.6492919922, 1535.8363037109, 11.120292663574, 270.00003051758},
+	{557, 2024.8865966797, 1526.9187011719, 11.120292663574, 270.00003051758},
+	{402, 2024.3311767578, 1519.9007568359, 10.760312080383, 270.00003051758},
+	{568, 2024.1086425781, 1515.0710449219, 10.79531288147, 270.00003051758},
+	{457, 2024.5035400391, 1510.8114013672, 10.549660682678, 270.00003051758},
+	{475, 2024.2908935547, 1506.1636962891, 10.72031211853, 270.00003051758},
+	{541, 2024.7091064453, 1499.3103027344, 10.520312309265, 270.00003051758},
+	{429, 2025.9349365234, 1490.2434082031, 10.5703125, 270.00003051758},
+	{429, 2025.6474609375, 1494.671875, 10.5703125, 270.00003051758},
+	{469, 2024.5600585938, 1623.0500488281, 12.002040863037, 270.00003051758}
+}
```

Comment:
Missing newline at the end of the file.
```suggestion
}

```